i6_models/parts/blstm.py

```diff
@@ -0,0 +1,84 @@
+from dataclasses import dataclass
+import torch
+from torch import nn
+
+from i6_models.config import ModelConfiguration
+
+
+@dataclass
+class BlstmEncoderV1Config(ModelConfiguration):
+    """
+    Attributes:
+        num_layers: number of bi-directional LSTM layers, minimum 2
```

Comment:
nitpick: you say minimum 2 here, but I don't see this being enforced anywhere and I'm also not sure if it couldn't just as well work with 1.

Reply:
Ops, that was from an older version.